src/decoders/grades-overview.ts

```diff
@@ -25,6 +25,7 @@ export const buildOverview = (data: any): GradesOverview => {
           isChildSubject: subject.sousMatiere,
           // TODO
           color: "string",
+          coefficient: subject.coef,
```

Comment:
```suggestion
          coefficient: Number(subject.coef),
```

d'après `grade.ts` :  https://github.com/LiterateInk/Pawdirecte/blob/b70d595340e2e261c73975e2416ca95e5af48bac/src/decoders/grade.ts#L11